NDTensors/test/dense.jl

```diff
@@ -25,83 +26,91 @@ using Test
 
       randn!(A)
 
-      for I in eachindex(A)
-        @test A[I] != 0
-      end
-
-      for I in eachindex(A)
-        @test A[I] != 0
-      end
-
       @test ndims(A) == 2
       @test dims(A) == (3, 4)
       @test inds(A) == (3, 4)
 
-      A[1, 1] = 11
+      Aview = A[2:3, 2:3]
+      @test dims(Aview) == (2, 2)
 
-      @test A[1, 1] == 11
+      B = dev(Tensor(undef, (3, 4)))
+      randn!(B)
+      C = copy(A)
+      C = permutedims!!(C, B, (1, 2), .+)
 
-      Aview = A[2:3, 2:3]
+      Ap = permutedims(A, (2, 1))
+      A
+      Ap
```

Comment:
?

Reply:
These were a relic of debugging, I have removed them thanks!